<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QJson2CppStudioClass</class>
 <widget class="QWidget" name="QJson2CppStudioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>118</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QJson2CppStudio</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leInFile"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输出文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leOutFile"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbIsClass">
       <property name="text">
        <string>类/结构体</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leObjName">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConv">
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QJson2CppStudio.qrc"/>
 </resources>
 <connections/>
</ui>
